src/fw/wscript

```diff
@@ -119,8 +119,7 @@ def _generate_memory_layout(bld):
         flash_size = '2048K'
 
     elif bld.env.MICRO_FAMILY == 'NRF52840':
-        #bootloader_size = '64K'
-        bootloader_size = '156K' # softdevice takes room
+        bootloader_size = '0K'
```

Comment:
probably leave this as an XXX because we will have to put the bootloader in there

Reply:
done